src/app/app.component.ts

```diff
@@ -6,20 +6,34 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatIconModule } from '@angular/material/icon';
 import { MatToolbarModule } from '@angular/material/toolbar';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import {MatSelectModule} from '@angular/material/select';
 
 @Component({
   selector: 'app-root',
   standalone: true,
-  imports: [RouterOutlet, MatButtonModule, MatSidenavModule, MatIconModule, MatToolbarModule, RouterLink],
+  imports: [RouterOutlet, MatButtonModule, TranslateModule, MatSelectModule, MatSidenavModule, MatIconModule, MatToolbarModule, RouterLink],
   templateUrl: './app.component.html',
   styleUrl: './app.component.scss'
 })
 export class AppComponent implements OnDestroy, OnInit {
   private destroy$: Subject<void> = new Subject<void>();
   
   authService = inject(AuthService);
+
+  translate = inject(TranslateService)
   
-  constructor (private router: Router) {}
+  constructor (private router: Router) {
+    this.translate.addLangs(['en', 'de', 'lv']);
+
```

Comment:
would be nice if you could extract logic related to translations to app.service.ts :)